tests/chainer_tests/training_tests/extensions_tests/test_plot_report.py

```diff
@@ -23,5 +23,14 @@ def test_available(self):
         else:
             self.assertEqual(len(w), 1)
 
+    def test_lazy_import(self):
+        if extensions.PlotReport.available():
```

Comment:
Isn't it possible to write as `@unittest.skipUnless(extensions.PlotReport.available(), 'matplotlib is not installed')`